### PR TITLE
update lv prometheus viewer project

### PR DESCRIPTION
**Pull request by JimCAtSB**
Address CQS-5255, CQS-5256, CQS-5257.

Remove shaded library use
remove execute bit on sources

### CODE REVIEW

File: components/lv-prometheus-viewer/sb-lv-prometheus/src/main/eventflow/com/tibco/ep/community/components/sblvprometheus/GetSelectMetrics.sbapp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<modify version="10.6.1_0a0fdf7f8f3d4f25851d53e0e55c97ce2ece3d22">
+<modify version="10.6.2-SNAPSHOT_06c6e0cfbdff659bfd1712330429035e8ee1303d">
```
Comment (TIBCOguillonapa) [inline components/lv-prometheus-viewer/sb-lv-prometheus/src/main/eventflow/com/tibco/ep/community/components/sblvprometheus/GetSelectMetrics.sbapp:2]: This should still be 10.6.1 (latest available version).

**Review by TIBCOguillonapa — CHANGES_REQUESTED** [review]
There's a single comment about the version used in the EventFlow. I'll leave the actual functionality of the sample for you to verify Jim. Thanks!

**Comment by JimCAtSB** [comment]
Sorry, that slipped by me. Fixed now. Thanks, Jim...

**Review by TIBCOguillonapa — APPROVED** [review]
Looks good to me!

**Pull request merged**